pipelinewise/cli/utils.py

```diff
@@ -343,11 +349,24 @@ def search_files(search_dir, patterns=None, sort=False, abs_path=False):
         # Search files and sort if required
         p_files = []
         for pattern in patterns:
-            p_files.extend(
-                filter(os.path.isfile, glob.glob(os.path.join(search_dir, pattern)))
-            )
+            p_files.extend(glob.glob(os.path.join(search_dir, pattern)))
         if sort:
-            p_files.sort(key=os.path.getmtime, reverse=True)
+            # We will extract the time of the run from the name
+            # of the log file.
+            timestamp_pattern = re.compile(
```

Comment:
could we maybe unify the parsing logic here with the log attributes / timestamp extraction from `cli.utils.extract_log_attributes`?

Reply:
Good spot - missed that there was already logic for this!